bokeh/sphinxext/bokeh_plot.py

```diff
@@ -147,6 +147,14 @@ def visit_html(visitor, node):
 
     html = visit_html.__func__, None
 
+class sampledata_node(nodes.Admonition, nodes.Element):
+    pass
+
+def visit_sampledate_node(self, node):
+    self.visit_admonition(node)
+
+def depart_sampledate_node(self, node):
+    self.depart_admonition(node)
```

Comment:
I tried to get rid of the Admition part, because I think I don't need this, but all tests were unsuccessful.